rollbar/__init__.py

```diff
@@ -1007,6 +1014,34 @@ def _build_bottle_request_data(request):
 
     return request_data
 
+
+def _build_wsgi_request_data(request):
+    request_data = {
+        'url': wsgiref.util.request_uri(request),
+        'user_ip': _wsgi_extract_user_ip(request),
+        'method': request.get('REQUEST_METHOD'),
+    }
+    if 'QUERY_STRING' in request:
+        request_data['GET'] = urlparse.parse_qs(request['QUERY_STRING'], keep_blank_values=True)
+        # Collapse single item arrays
+        request_data['GET'] = { k: v[0] if len(v) == 1 else v for k, v in request_data['GET'].iteritems()}
```

Comment:
Should probably switch to `items()` for python 3 compat (here and elsewhere). See the failing tests: https://travis-ci.org/rollbar/pyrollbar/builds/53403146